WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3565,17 +3567,18 @@ e_byzantium = {
 		capital = 560 # Cherson
 		
 		hellenic_pagan = 600
-		bosporan_faith = 200
+		bosporan_faith = 1000
 		
 		allow = {
-			hidden_tooltip = {
+			OR = {
+				religion = bosporan_faith
```

Comment:
Wrong. As it stood, Bosporans _could not create the Kingdom where their religion began_. I created a work around to stupid, arbitrary requirement that limited its creation. The only reason I didn't remove it was because I didn't want to hear any of your lip.